assets/sass/components/_cards.scss

```diff
@@ -30,6 +32,32 @@
     }
 }
 
+/* =========================================================================
+   Headed Card
+   ========================================================================= */
+.card--headed {
```

Comment:
`card--capped` feels maybe?. IDK let's leave it and keep an eye on the usage.

Reply:
Personally if this usage is pretty much just for the logos I'd call this `logo-card` and not have the abstraction, but if we use it more it could do with a clearer name.

Reply:
I've just merged the overlapping styles into `.logo-card` and removed the modifier from `.card` – have another look?